launcher/service/fake_attestationverifier.go

```diff
@@ -0,0 +1,79 @@
+// Package service is a fake implementation of the AttestationVerifier for testing.
+package service
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+
+	servpb "github.com/google/go-tpm-tools/launcher/proto/attestation_verifier/v0"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	_ "embed"
+)
+
+// FakeToken is generated by fake_tokens/fake_rsa_token.txt.
+//go:embed fake_tokens/fake_rsa_token.txt
+var FakeToken []byte
+
+// FakeServer implements the AttestationVerifier service methods. The initial
+// connection ID produced by the server will be "0", incrementing with every
+// subsequent request to GetParams.
+type FakeServer struct {
+	// conns maps connection IDs to nonces.
+	conns map[string][]byte
+
+	// nextConnID represents the next connection ID the server will produce.
+	nextConnID int
+}
+
+// Check that FakeServer implements servpb.AttestationVerifierServer.
+var _ servpb.AttestationVerifierServer = &FakeServer{}
+
+// New constructs a new FakeServer.
+func New() FakeServer {
+	fs := FakeServer{}
+	fs.conns = make(map[string][]byte)
+	fs.nextConnID = 0
+	return fs
+}
+
+func (s *FakeServer) GetParams(ctx context.Context, req *servpb.GetParamsRequest) (*servpb.GetParamsResponse, error) {
+	nonce := make([]byte, 32)
+	rand.Read(nonce)
+
+	connID := fmt.Sprint(s.nextConnID)
```

Comment:
string(s.nextConnID) => fmt.Sprint(s.nextConnID)

Reply:
what's the reasoning here?

Reply:
I am getting a warning `conversion from int to string yields a string of one rune, not a string of digits`
I suppose `connID` should be a string of digits?

Reply:
ah of course it's casting as a char. makes sense, thanks!